gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -741,4 +741,7 @@ private String safeGetMessage(Throwable t) {
     return message != null ? message : "null";
   }
 
+  void setTokenLimitPerUser(int tokenLimitPerUser) { // visible for testing
```

Comment:
You could also use reflection in the test to set this, rather than adding a method ONLY for testing.

Reply:
I'm not a fan of any of those (adding the accessor / using reflection) but thought this way is a little bit better. With reflection I need to reference the field name as a String which is not refactoring friendly.
I'll see if I can reconfigure the mocks and call the init() on the TokenResource again to change the limit.

Reply:
I managed to change the limit without the accessor method or reflection.